File: backend/src/main/java/capstone/rt04/retailbackend/request/refund/RefundLineItemRequest.java
package capstone.rt04.retailbackend.request.refund;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class RefundLineItemRequest {
    private Long transactionLineItemId;
    private Integer quantityToRefund;
    private Long staffId;
}

File: frontend/customer-web-app/src/models/refund/RefundLineItemRequest.js
class RefundLineItemRequest {
  transactionLineItemId;
  quantityToRefund;
  staffId;

  constructor(transactionLineItemId, quantityToRefund, staffId) {
    this.transactionLineItemId = transactionLineItemId;
    this.quantityToRefund = quantityToRefund;
    this.staffId = staffId;
  }
}

export default RefundLineItemRequest;
